solution/zenkoversion.yaml

```diff
@@ -50,8 +50,8 @@ spec:
       image: kafka
       tag: 2.12-2.3.0
     vault:
-      image: vault
-      tag: zenko-2.0.0
+      image: ${SOLUTION_REGISTRY}/vault
```

Comment:
```suggestion
      image: vault
```

Reply:
I was following what zenko-ui has on line # 11 of the same file.
and the same is mentioned in the summary of this ticket: https://scality.atlassian.net/browse/ZENKO-3134

If I specify just vault, won't it look for it in dockerhub and not solution registry?

CC 
@tcarmet for confirmation as he reported the ticket.
@nicolas2bert for awareness